reiscout/reiscout-com
# redmine task #704534 implementation

To deploy:
* pull in the code
• install modules one by one reiscout_map_feature -> reiscout_map -> reiscout_map_atlanta_content_feature , the content one as a last one after cache clearing
• probably you will need manually create Atlanta Map node
• probably you will need manually create Atlanta Map product with SKU 'all-atlanta-map-squares'
• enable the deploy module
• set price 10 p for square product on admin/config/reiscout-points/products-prices
• set price 1000 p for Atlanta product on admin/config/reiscout-points/products-prices
• enable points currency
• check is private filesystem set up
• cd to scripts and run the script from scripts dir to import squares from archive (pass squares dir path relative to docroot)
* uuid in reiscout_map_node_view() should be the one the Atlanta map node.

To test:
* create map square node with route files
* check that corresponding product created
* check that non-author and non admin has no access for file fields and files by their uri's 
* purchase the individual square
* check that you have access now and created order is fine
* check that after purchasing Atlanta map product you have access for all the squares now
